akvo/rsr/tests/models/test_project.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+# Akvo Reporting is covered by the GNU Affero General Public License.
+# See more details in the license.txt file located at the root folder of the Akvo RSR module.
+# For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+
+
+from unittest import TestCase
+
+from django.contrib.auth import get_user_model
+
+from akvo.rsr.models import Project
+from akvo.rsr.models import ProjectUpdate, PayPalGateway, MollieGateway, PaymentGatewaySelector
+
+
+class ProjectModelTestCase(TestCase):
+    """Tests for the project model"""
+
+    def test_project_last_update(self):
+        """ Test Project.last_update. The field is a denormalization keeping track of the latest
```

Comment:
When running the tests in the verbose mode, the first line of the doc string is printed out.  It would be nice to have a summary line, followed by an explanation ([as described here](https://www.python.org/dev/peps/pep-0008/#documentation-strings)).